src/sentry/runner/commands/cleanup.py

```diff
@@ -132,16 +129,21 @@ def multiprocess_worker(task_queue: _WorkQueue) -> None:
                 )
 
                 while True:
+                    debug_output(f"Processing chunk of {len(chunk)} {model_name} objects")
+                    metrics.incr(
+                        "cleanup.chunk_processed", tags={"model": model_name}, amount=len(chunk)
```

Comment:
I want to have a sense of things working when I run the custom script. I'm not using a sample rate of 1 unlike other metrics.